### DISCUSSION TITLE
no response from the stream

### DISCUSSION THREAD

**Discussion by chethan-b-hpe** [nats-io/nats-server#5804] [General]
Hello here,recently we were running load test on nats, but intermittently getting no response from the stream while publishing the messages to jetstream , here is the piece of the code for publishing messages.
we are creating the stream at the startup of the application itself and using persistent volumes for nats also, So I don't think there is issue from the code

`	for _, subjectObj := range subject {

		_, err = js.Publish(ctx, subjectObj, eventData)
		if err != nil {
			logger.Errorf("Error publishing event to subject %s: %v", subjectObj, err)
			return nil, err
		}
	}`

**Comment by wallyqs** [comment]
What is the configuration of the stream?

**Reply by chethan-b-hpe** [reply]
here is the config :
`
_, err = js.CreateOrUpdateStream(ctx, jetstream.StreamConfig{
		Name:        "events",
		Description: "Webhook Events",
		Subjects: []string{
			"events.>",
		},
		MaxBytes:  1024 * 1024 * 1024,
		Replicas:  serverConfig.Replicas,
		Retention: jetstream.WorkQueuePolicy,
		Storage:   jetstream.FileStorage,
	})
`

**Reply by wallyqs** [reply]
replicas is 3?

**Reply by chethan-b-hpe** [reply]
yes its 3

**Reply by darwin67** [reply]
seeing similar issues with `PublishAsync` as well.

interestingly enough, it's as if the leader of the stream got into a weird state, and manually triggering a new leader election stopped the error.
the only thing that seems suspicious is the slow consumer count was non 0 on the previous leader.
